### DISCUSSION TITLE
Checking prior  and current values in validation rules

### DISCUSSION THREAD

**Discussion by DamienUKCEH** [netbox-community/netbox#18172] [Help Wanted!]
hi all

am trying to create validation rules that force business process
what am trying to do  is check that devices  and Vms have a state of planned before going active 

how can I check that the prior status was set to "Planned " ?

**Comment by DamienUKCEH** [comment]
has notbody else tryied mapping a business process into valudation rules to enforce a work flow ?

**Comment by DamienUKCEH** [comment]
@cruse1977 is there any documentation on how to implement what would be triggers on in SQL land ?

**Comment by cruse1977** [comment]
Hi, You may be able to do this via a custom validator and using the instance._prechange_snapshot field to check the value of status pre the change being applied.
